<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Learn</class>
 <widget class="QWidget" name="Learn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>615</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>392</width>
    <height>615</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>392</width>
    <height>615</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <widget class="QPushButton" name="starWord">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(78,78,78);
    background-color: rgba(255, 255, 255, 0);
	font-size:16px;}
/*
MyCheckBox:checked{
	font-size:7px;
    color:rgb(255, 170, 0);
    background-color: rgba(255, 255, 255, 0);
	border-radius:0px;
}
MyCheckBox::indicator{
	width: 0px;
	height: 0px;
}*/</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteWord">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(78, 78, 78);
background-color: rgba(255, 255, 255, 0);
font-size:15px;


/*

}*/
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:16px;
QPushButton{
	//background-color: #EB5353;
	color: rgb(100, 35, 35);
	border-radius:6px;
}
QPushButton:hover {
	font-size:7px;
	color: #EB5353;
	//background-color: #EB5353;
	border-radius:6px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>301</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="selections">
    <item>
     <widget class="QPushButton" name="selection_1">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
background-color: rgba(78, 78, 78, 120);
border-radius: 10;
</string>
      </property>
      <property name="text">
       <string>Selection1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selection_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
background-color: rgba(78, 78, 78, 120);
border-radius: 10;</string>
      </property>
      <property name="text">
       <string>Selection1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selection_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
background-color: rgba(78, 78, 78, 120);
border-radius: 10;</string>
      </property>
      <property name="text">
       <string>Selection1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selection_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
background-color: rgba(78, 78, 78, 120);
border-radius: 10;</string>
      </property>
      <property name="text">
       <string>Selection1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="wordLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);
background-color: rgba(78, 78, 78, 0);
border-radius: 10;
qproperty-alignment: 'AlignLeft';
qproperty-wordWrap: true;
font-size: 45px</string>
   </property>
   <property name="text">
    <string>Word</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
